lms/djangoapps/mobile_api/settings.py

```diff
@@ -0,0 +1,7 @@
+'''
+    Settings for Mobile Api
+'''
+
+FB_SETTINGS = {'_FACEBOOK_APP_SECRET': "6c26348ef355f53a531890e980ddc731",
+    '_FACEBOOK_APP_ID': "735343629893573",
+    '_FACEBOOK_API_VERSION': "v2.2"}
```

Comment:
These settings have to move to https://github.com/edx/edx-platform/blob/master/lms/envs/common.py and https://github.com/edx/edx-platform/blob/master/lms/envs/aws.py.

They need to be overridden by our devOps via JSON configuration files.

You can look for how ZENDESK_API_KEY is implemented in the above files as an example.